src/demos/select-field/select-field-demo.component.ts

```diff
@@ -79,4 +113,29 @@ export class SkySelectFieldDemoComponent implements OnInit {
     this.fruitStream.next(this.fruits);
     this.colorStream.next(this.colors);
   }
+
+  private showCustomPicker(
+    pickerType: 'color' | 'fruit',
+    pickerContext: SkySelectFieldPickerContext,
+    setValue: (value: SkySelectField[]) => void
+  ) {
+    const picker = this.modal.open(
+      SkySelectFieldDemoCustomPickerComponent,
+      [
+        {
+          provide: SkySelectFieldDemoCustomPickerContext,
+          useValue: {
+            pickerContext,
+            pickerType
+          }
+        }
+      ]
+    );
+
+    picker.closed.subscribe((args: SkyModalCloseArgs) => {
+      if (args.reason === 'save') {
```

Comment:
Should we be covering the "closed" route as well? Here's what I was running into:
 
1. Open the multiselect picker
2. Make some selections
3. Click 'close' (NOT select).
4. Notice tokens are not updated, which is expected.
5. Open the multiselect picker again
6. Notice your old selections are still selected.